.github/workflows/workflow_finetune.yml

```diff
@@ -96,6 +97,10 @@ jobs:
                   result['General']["gpt_base_model"] = True
               else:
                   result['General']["gpt_base_model"] = False
+              if "${{ matrix.model }}" == "mistralai/Mistral-7B-v0.1":
+                  result['General']['lora_config']['target_modules'] = ["q_proj", "k_proj", "v_proj", "o_proj", "gate_proj", "up_proj", "down_proj", "lm_head",]
```

Comment:
This should be added in `Run PEFT-LoRA Test` instead of this `Run Finetune Test`

Reply:
updated